calibrators/marker_radar_lidar_calibrator/README.md

```diff
@@ -0,0 +1,168 @@
+
+The images below illustrate the process of radar background model construction and radar foreground extraction that is described in Step 1 and Step 2.
+The blue 3d voxels, which are shown in a 2D grid in the images, are denoted as the background voxel if radar objects are in the voxels during the background model construction. Once the background model is constructed, it becomes straightforward to extract the foreground points in the calibration area. For the lidar, the background model and foreground extraction process are the same as the radar process described above.
+
+<table>
+  <tr>
+    <td><img src="../docs/images/marker_radar_lidar_calibrator/background_construction.svg" alt="background_construction" width = 700px height = 300px ></td>
+    <td><img src="../docs/images/marker_radar_lidar_calibrator/foreground_extraction.svg" alt="foreground_extraction" width = 700px height = 300px ></td>
+   </tr>
+   <tr>
+    <td><p style="text-align: center;">Background model construction.</p></td>
+    <td><p style="text-align: center;">Foreground extraction</p></td>
+  </tr>
+</table>
+
+### Step 3: Matching and filtering
+
+Since it is not possible to directly differentiate reflector detections, whether there are multiple targets in the calibration area or if the detections are from humans or radar reflectors, we rely on the initial calibration to pair each lidar detection with its closest radar detection, and vice versa. A detection pair is accepted if they are mutually their closest matches. Once a match is made, it is evaluated against existing hypotheses (monitored by a Kalman filter): if it aligns with an existing hypothesis, that hypothesis is updated; if it does not align with any, a new hypothesis is created. When a hypothesis achieves convergence, it is added to the calibration list.
+
+### Step 4: Calibration
+
+After matching detection pairs from the sensors, we can compute the transformation between them using rigid transformation estimation algorithms. Currently, we support two methods: a 2d SVD-based approach and a yaw-only rotation approach.
+
+For the 2d SVD-based method, since radar detections lack a z component, we reduce the problem to 2d by setting the z component of lidar detections to zero. We then estimate the rigid transformation using the SVD-based method provided by PCL, which leverages SVD to find the optimal rotation component and then calculates the translation component based on the rotation.
+
+The yaw-only rotation method, on the other hand, calculates the average yaw angle difference of all pairs and estimates the transformation, considering only yaw rotation, between the sensors. Generally, the 2d SVD-based method is preferred when valid; otherwise, the yaw-only rotation method is used as the calibration output.
```

Comment:
extra comma

Reply:
`considering only yaw rotation, between the sensors`
Actually this is not true right? Is that not the parallel frame?

Reply:
Extra comma means there is an extra comma
Missing comma means there is a missing comma

In this case, the one I am referring to is "yaw rotation, between the sensors"
Remember that grammarly does not have context, nor knowledge about the topic. As such, the most you get is a grammarly correct text. Not one that expresses what you wan to to convey

Also, `yaw angle between sensors` does not really exist with no more explanation and I think this was written somewhere else (think well what do you need to uniquely define a yaw)

Reply:
Thanks, I rewrite the whole paragraph in https://github.com/tier4/CalibrationTools/pull/164/commits/4f7a533dae98254e0ffcc87c385fe38e25d5fd4e.

Reply:
Why the base link makes for an appropriate radar parallel frame is not explained

Reply:
Thanks, fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/443a4b57d1a807abdbb480e971667de2e536a0e3

Reply:
There are both:
`radar parallel` frame 
and
`radar parallel frame`

Reply:
The radar parallel does not have ...
It is designed to have ...
or
it defined as ...

Reply:
You make it sound like just transforming points to the radar parallel frame make the points have z=0

Reply:
fix them in https://github.com/tier4/CalibrationTools/pull/164/commits/944dc88e11b72c6338497ec4036dededc2ec8dde